File: divvy.py
import urllib2
import json

json_data = urllib2.urlopen("http://divvybikes.com/stations/json")
data = json.load(json_data)

stations = data["stationBeanList"]

for station in stations:
	loc = station["stationName"]
	ava = station["availableDocks"]
	print "There are " + str(ava) + " docks available at " + loc

File: divvy.pl
#!/usr/bin/perl
use strict;
use warnings;
use LWP::Simple;
use JSON qw( decode_json );

my $data = decode_json(get("http://divvybikes.com/stations/json"));
my $stations = $data->{"stationBeanList"};

foreach my $station(@$stations) {
	print "There are ".$station->{"availableDocks"}." of ".$station->{"totalDocks"}." docks available at ".$station->{"stationName"}."\n";
}

print "Last update: ".$data->{"executionTime"}."\n";
